griptape/tools/aws_s3_client/tool.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+import boto3
+from schema import Schema, Literal
+from attr import define, field, Factory
+from griptape.artifacts import BaseArtifact, TextArtifact, ErrorArtifact
+from griptape.core.decorators import activity
+from griptape.tools import BaseAwsClient
+
+@define
+class AwsS3Client(BaseAwsClient):
+    session: boto3.session = field(kw_only=True)
+    s3_client: boto3.client = field(
+        default=Factory(lambda self: self.session.client("s3"), takes_self=True)
+    )
+
+    @activity(config={
+        "description": "Retruns the access control list(ACL) of an bucket.",
```

Comment:
Change to "Can be used to get an access control list (ACL) of an AWS S3 bucket."

Remember, the LLM has no context about this tool outside of the activity description, so the description should always be comprehensive.